### ISSUE TITLE
Reso 98 bug.

### ISSUE THREAD

**Issue by Altru1sticTest** [Yellow-Dog-Man/Resonite-Issues#2900]
### Describe the bug?

When I spawn the "Voice Options 98'" facet it just spawns the default voice option facet.

### To Reproduce

Open the "Resonite Essentials" folder then, open the "Facets" Folder than the "Reso 98'" folder and click on the "Voice Options 98'" facet and press the spawn facet button near the top of the menu window.

### Expected behavior

It would spawn the proper voice options facet.

### Resonite Version Number

2024.9.6.1189

### What Platforms does this occur on?

Windows

### What headset if any do you use?

Meta Quest 3

### Log Files

[2024.5.31.202 - 2024-06-04 15_46_35.log](https://github.com/user-attachments/files/16919276/2024.5.31.202.-.2024-06-04.15_46_35.log)

### Reporters

Altruist

**Comment by Zyro1331** [comment]
This is likely due to the Reso98 Voice options facet still containing the FacetPreset component on them, which causes it to be replaced with the latest official facet instead of the stylized one.
 
@shiftyscales This issue should probably be relabeled and directed towards the Content team instead

**Comment by shiftyscales** [comment]
I agree, it sounds like this facet in the essentials didn't get updated properly to remove the preset component.

Assigning @AegisTheWolf.

**Comment by AegisTheWolf** [comment]
All Reso 98 facets are in the process of being reworked, unified with its setup and respecting aspect ratios as much as possible for each facet. The first ones to be reworked are the top bar facets, the voice facets are currently being worked on specifically, I will move the old ones into a folder in the Reso98 folder, and upload the new ones when there done. (this issue will be closed when the new top bar facets are done)

**Comment by AegisTheWolf** [comment]
Still having an odd issue with the Volume Level facet as the Mesh that displays the audio waveform randomly disappears and reappears but otherwise all of the top row facets have been reworked, Im going to create Reso98 versions of the newer battery level facets next. (and icon variants of each too)

**Comment by AegisTheWolf** [comment]
Ah forgot to close this issue as new facets were created and updated to include the new separated voice options and levels facets, they are available in the Essentials now
